samples/kaspresso-sample/src/androidTest/kotlin/com/kaspersky/kaspressample/docloc_tests/ChangeSysLanguageTestCase.kt

```diff
@@ -29,16 +31,20 @@ class ChangeSysLanguageTestCase : DocLocScreenshotTestCase(
     val activityTestRule = ActivityTestRule(MainActivity::class.java, true, false)
 
     @Test
-    fun showAndroidSettings() = run {
-        step("Start Android OS Settings") {
-            val intent = Intent(Settings.ACTION_SETTINGS)
-            intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
-            device.targetContext.startActivity(intent)
-            Thread.sleep(3000)
-        }
+    fun showAndroidSettings() {
+        Assume.assumeTrue(Build.VERSION.SDK_INT <= Build.VERSION_CODES.O) // Can't change system locale on newer versions
```

Comment:
We should fix it or describe this limitation in the docs

Reply:
Maybe throw an exception too?